pkg/edgeview/src/tcp.go

```diff
@@ -75,6 +77,11 @@ const (
 	tcpMaxMappingNUM  int           = 5
 	tcpIdleTimeoutSec float64       = 1800.0
 	tcpCheckTimeout   time.Duration = 300 * time.Second
+	tcpKubeEndpoint   string        = "localhost:6443"
+	kubeYamlFile      string        = "/run/.kube/k3s/user.yaml"
```

Comment:
Can you base it on:
```
./kubeapi/kubeapi.go:	EVEkubeConfigFile = "/run/.kube/k3s/k3s.yaml"
``` ?
This would make it easier to change it from `/run/.kube` to `/run/kube` in case we find out that it is inconvenient when debugging to have hidden directories in `/run`.

Reply:
this directory mainly have some secret in the config files, since other PRs also uses this way. We can change later if we all agree.